src/resources/js/Pages/Employee/Logs/Index.vue

```diff
@@ -43,24 +43,23 @@
           </h2>
 
           <div class="relative pv2 ph2 bb bb-gray">
-            <input type="hidden" name="url" id="url" class="form-control" :value="url">
```

Comment:
Unless you're submitting a form, hidden inputs are another good indicator that there is a better way to do things in a reactive language like vue or react